packages/spacecat-shared-rum-api-client/src/functions/form-vitals.js

```diff
@@ -31,7 +33,8 @@ function initializeResult(url) {
 
 const metricFns = {
   formview: (bundle) => {
-    const formView = bundle.events.find((e) => e.checkpoint === 'viewblock' && FORM_SOURCE.includes(e.source));
+    const formView = bundle.events.find((e) => e.checkpoint === 'viewblock'
```

Comment:
Pls handle the cases where RUM event describes the type of form (like search,login)